inspektor/commands/github.py

```diff
@@ -10,10 +10,56 @@
 # See LICENSE for more details.
 
 import logging
+import os
 
 from cliff.command import Command
 
-from inspektor.patch import GithubPatchChecker
+from inspektor import utils
+from inspektor.patch import PatchChecker
+
+
+class GithubPatchChecker(PatchChecker):
+
+    def __init__(self, args, patch=None, logger=logging.getLogger('')):
+        super(GithubPatchChecker, self).__init__(args=args, patch=patch,
+                                                 logger=logger)
+        self.github_id = args.gh_id
+        self.github_parent_project = args.parent_project
+        assert self.github_id is not None
+        assert self.github_parent_project is not None
+        self.patch = self._fetch_from_github()
+
+    def _get_github_repo_name(self):
+        return self.vcs.get_repo_name()
+
+    def _get_github_url(self):
+        """
+        Gets the correct github URL for the given project.
+        """
+        repo = self._get_github_repo_name()
+        return ("https://github.com/%s/%s/pull/%s.patch" %
```

Comment:
Btw adding feature like this might bring back inspector into the game (checking per-commit changes, ideally along with custom command for functional tests). Anyway as I said I'm not sure it's worth the effort. I'd be actually fine deprecating this feature, unless you know of people using it.